File: FouFouFood_ServeurNodeJS/foufoufood_serveur/Restaurant.js
class Restaurant {
    constructor(id, name, address, cuisine, phone, openingHours, menu, rating, reviews) {
        this.id = id;
        this.name = name;
        this.address = address;
        this.cuisine = cuisine;
        this.phone = phone;
        this.openingHours = openingHours;
        this.menu = menu;
        this.rating = rating;
        this.reviews = reviews;
    }

    addMenuItem(menuItem) {
        this.menu.push(menuItem);
    }
}

module.exports = Restaurant;

File: foufoufood_client_android/app/src/main/java/com/example/foufoufood/Restaurant.kt
package com.example.foufoufood

class Restaurant(
    val id: Int,
    val name: String,
    val address: String,
    val cuisine: String,
    var phone: String,
    var openingHours: String,
    val menu: List<Int>,
    var rating: Float,
    val reviews: List<String>
)


